awscli/examples/schemas/delete-discoverer.rst

```diff
@@ -0,0 +1,8 @@
+**To delete a discoverer**
+
+The following ``delete-discoverer`` deletes a discoverer. If the command succeeds, no output is returned. ::
```

Comment:
```suggestion
The following ``delete-discoverer`` deletes a discoverer. ::
```

For commands that produce no output, use the following line after the command and update all your examples to follow this format:

``This command produces no output.``

https://aws.github.io/aws-cli/docs_styleguide.html#command-produces-no-output